### PR TITLE
Add Symfony Flex support

### LINKED ISSUE #138: Symfony flex support
- [x] `web/app_*.php` -> `public/index.php`(ansible nginx, deploy)
- [x] `/var/logs`-> `/var/log`
- [x] Drop `/var/sessions`(ansible files)
- [x] Env (`SYMFONY_ENV` -> `APP_ENV`)

### PR DESCRIPTION
**Pull request by benji07**
close #138

### CODE REVIEW

File: src/Resources/envs/symfony/ansible/group_vars/app.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ app_patterns:
   ###############
 
   environment_variables:
-    - SYMFONY_ENV:                 "{{ app.env }}"
     - APP_SECRET:                  ThisTokenIsNotSoSecretChangeIt
```
Comment (ogizanagi) [inline src/Resources/envs/symfony/ansible/group_vars/app.yml]: Tout ça est à virer également non ?

File: src/Resources/envs/custom/ansible/group_vars/app.yml [hunk 2]
```diff
@@ -106,7 +106,7 @@ app_patterns:
   #  - file: app
   #    user: "{{ app.user }}"
   #    environment:
-  #      - SYMFONY_ENV: "{{ app.env }}"
```
Comment (ogizanagi) [inline src/Resources/envs/custom/ansible/group_vars/app.yml:109]: Je ne sais pas si on garde ça en tant qu'exemple ou non. En vérité je ne vois pas vraiment d'usage aux var d'env en dev. Les `.env` font référence contrairement à du staging/prod.

File: src/Resources/envs/symfony/ansible/group_vars/app.yml [hunk 3]
```diff
@@ -90,15 +86,15 @@ app_patterns:
...
           - error_log:   "{{ app.log_dir }}/nginx.error.log"
           - include:     conf.d/app_gzip
           - location /:
-            - try_files: $uri /app.php$is_args$args
-          - location ~ ^/(app(_[-\w]+)?)\.php(/|$):
+            - try_files: $uri /index.php$is_args$args
+          - location ~ ^/index\.php(/|$):
             - include: conf.d/app_php_fpm
-            #- internal;
+            - internal;
```
Comment (ogizanagi) [inline src/Resources/envs/symfony/ansible/group_vars/app.yml:92]: 🔥 :trollface:

File: src/Resources/envs/symfony/ansible/group_vars/app.yml (hunk 1 above)
Comment (chalasr) [reply src/Resources/envs/symfony/ansible/group_vars/app.yml]: 👍

**Comment by NicolasDievart** [comment]
Does it mean Manalize will only work out of the box for symfony flex project (4.x branch) and not for Symfony 3.4 (which is the lts version) ?

**Comment by ogizanagi** [comment]
@NicolasDievart : You can already use Flex with 3.4 projects and it's actually the recommended way.
Also keep in mind Manalize is for new projects init (but you can also follow changes through the diff command). So it's not an issue for old projects still using the symfony-standard edition skeleton.

**Comment by NicolasDievart** [comment]
@ogizanagi Yes, sorry, removed my comment after consulting the doc that said to introduce flex as soon as 3.3, my bad

**Comment by ogizanagi** [comment]
Thanks @benji07.

**Pull request merged**